### PR TITLE
Spacesuits won't show 'you turn off x' if it's turned off automatically

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Spacesuits won't show 'you turn off x' if it's turned off automatically, either from removing the power cell or it losing power.
## Why It's Good For The Game

free gbp

also more word better more accurate
## Changelog
:cl:
fix: Spacesuits won't show 'you turn off x' if it's turned off automatically, either from removing the power cell or it losing power.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
Wtf my fix tag

File: code/modules/clothing/spacesuits/_spacesuits.dm [hunk 1]
```diff
@@ -107,13 +107,13 @@
...
 	if(!cell)
-		toggle_spacesuit(user)
+		toggle_spacesuit()
 		update_hud_icon(user)
 		return
 
 	// cell.use will return FALSE if charge is lower than THERMAL_REGULATOR_COST
 	if(!cell.use(THERMAL_REGULATOR_COST))
-		toggle_spacesuit(user)
+		toggle_spacesuit()
```
Comment (ShizCalev) [inline code/modules/clothing/spacesuits/_spacesuits.dm:116]: This shouldn't be a surprise, fix the message saying that YOU did it and make it say that it turned off automatically instead.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/_spacesuits.dm [hunk 2]
```diff
@@ -221,7 +221,7 @@
 	to_chat(user, span_notice("You [cell_cover_open ? "open" : "close"] the cell cover on \the [src]."))
 
 /// Toggle the space suit's thermal regulator status
-/obj/item/clothing/suit/space/proc/toggle_spacesuit(mob/toggler)
+/obj/item/clothing/suit/space/proc/toggle_spacesuit(mob/toggler, manual_toggle = TRUE)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/spacesuits/_spacesuits.dm:224]: https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md#documenting-a-proc

Comment (ExcessiveUseOfCobblestone) [reply code/modules/clothing/spacesuits/_spacesuits.dm:224]: ping me when its done at cobby#0002

Comment (carlarctg) [reply code/modules/clothing/spacesuits/_spacesuits.dm:224]: really dude

File: code/modules/clothing/spacesuits/_spacesuits.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/clothing/spacesuits/_spacesuits.dm]: fixd

**Pull request merged**